packages/lib-classifier/src/components/Classifier/components/Feedback/FeedbackModal.js

```diff
@@ -49,17 +49,21 @@ class FeedbackModal extends React.Component {
           <>
             <Box
               height='medium'
-              overflow='auto'
               width='medium'
             >
-              {showViewer && <FeedbackViewer />}
-              <ul>
-                {messages.map(message =>
-                  <li key={Math.random()}>
-                    {message}
-                  </li>
-                )}
-              </ul>
+              {showViewer && (
+                  <Box height='100%'>
+                    <FeedbackViewer />
+                  </Box>)}
+              <Box overflow='scroll'>
```

Comment:
Why don't we give this Box a fixed height if a viewer is displayed so the viewer can have more visual real estate? It's still very small... I arbitrarily picked 200px here (hopefully the markings go 100% if done properly in the code):

<img width="465" alt="Screen Shot 2019-05-16 at 3 19 13 PM" src="https://user-images.githubusercontent.com/5016731/57884376-13d29380-77ee-11e9-968b-ad53ad529d5a.png">